lib/processes/application/index.js

```diff
@@ -66,16 +66,24 @@ function getAppOptions(config) {
 
     var commandArgs = options.command.split(/[\s]+/g);
 
+    var env = {
+      NODE_ENV: config.get('launchEnv', 'test'),
+      PORT: options.port,
+      PATH: './node_modules/.bin:' + process.env.PATH,
+    };
+
+    while (commandArgs.length && commandArgs[0].match('=')) {
```

Comment:
This means that the command in the start script overrides the things above, I don't think that's intended behavior.